This file is part of MXE. See LICENSE.md for licensing information.

From 7798a70fa69dc4994cc25f82ec3e9b2b368d50fe Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Sun, 30 Aug 2015 14:46:10 +0200
Subject: [PATCH] MyString: include windows.h

Functions CharUpperA, CharPrev etc
---
 CPP/Common/MyString.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/CPP/Common/MyString.h b/CPP/Common/MyString.h
index 7db8a37..297986f 100755
--- a/CPP/Common/MyString.h
+++ b/CPP/Common/MyString.h
@@ -4,6 +4,7 @@
 #define __COMMON_STRING_H
 
 #include <string.h>
+#include <windows.h>
 
 #include "MyVector.h"
 
-- 
1.7.10.4

